idr0008-rohn-actinome/screenB/idr0008-screenB-bulkmap-config.yml

```diff
@@ -58,6 +58,12 @@ columns:
       columns:
       - name: siRNA Identifier
         include: yes
+        omitempty: no
+      - name: siRNA Identifier
```

Comment:
yes, I agree that is the case when there is only one of the columns in the csv, but in this screen both siRNA Identifier and siRNA Pool Identifier are in the csv.